scapy/contrib/cansocket_native.py

```diff
@@ -74,6 +79,47 @@ def __init__(self, channel=None, receive_own_messages=False,
         self.ins.bind((self.channel,))
         self.outs = self.ins
 
+    def command(self):
```

Comment:
all your command functions look like very redundant code. Could you try to not repeat yourself.